packages/mcp-server/src/server.ts

```diff
@@ -209,6 +212,9 @@ export async function configureServer({
                 name: `prompts/get ${prompt.name}`,
                 attributes: {
                   "mcp.prompt.name": prompt.name,
+                  ...(context.userAgent && {
```

Comment:
we'll remove this once sentry's core instrumentation does this at the mcp.server level